graphene_django/tests/test_types.py

```diff
@@ -416,8 +417,6 @@ class Meta:
                 model = ReporterModel
                 fields = ["first_name", "custom_field", "email"]
 
-    assert len(record) == 0
```

Comment:
Shouldn't we keep the assertion, to prove that despite the warning something gets returned?

Reply:
`pytest.warns` triggers a warning as passing `None` is deprecated, they explicitly state to use `warnings` instead

Reply:
Any chance we could keep the assertion still?

Reply:
`with warnings.catch_warnings()` does the assertion under the hood: https://docs.pytest.org/en/latest/how-to/capture-warnings.html#additional-use-cases-of-warnings-in-tests

Reply:
... if you add `warnings.simplefilter("error")` which i forgot, fixed